src/test/java/com/google/api/codegen/gapic/testdata/py/python_no_path_templates.baseline

```diff
@@ -1155,22 +1155,10 @@ import sys
 from google.api_core.protobuf_helpers import get_messages
 
 from example.proto import no_path_templates_messages_pb2
-from google.api import http_pb2
```

Comment:
Are these changes safe?

Reply:
They seemed to reference unreachable code, but I can double check with someone who knows the `types.py` files better. Perhaps @lukesneeringer ?

Reply:
I think you're right.